### ISSUE TITLE
VIRUS BEWARD 

### ISSUE THREAD

**Issue by akiii-afk** [hydy100/R3nzSkin#164]
Please for the good of yourself do not download this. This file is a virus, and not like the former renz one. I know because ALL my accounts got hacked after using this.

**Comment by mrlobaloba420** [comment]
> Please for the good of yourself do not download this. This file is a virus, and not like the former renz one. I know because ALL my accounts got hacked after using this.

pretty sure you didnt download the file here, maybe you downloaded it from other sources like youtube

**Comment by akiii-afk** [comment]
https://www.virustotal.com/gui/file/ed36d1edb27574e1c9481f2968791fc624b7e8d72338b6e1b5ec4c5c5b2df6c6?nocache=1

nah just check this out, so many viruses

**Comment by 0xJsoul** [comment]
😱😱😱😱

**Comment by akiii-afk** [comment]
I mean thats literally what the website is saying, that there are more than half of the virus scanners that flag your code as malicious malware.

**Comment by akiii-afk** [comment]
I did, it hacked my riot information, steam, and discord. Changed all my passwords and etc.

**Comment by Bullseye110** [comment]
> I did, it hacked my riot information, steam, and discord. Changed all my passwords and etc.

Man, stop lying, you downloaded it from somewhere else, this app is flagged on virustotal, because it's an injector/hacktool. Of course it will be flagged as virus as you try to use "paid" skins for free. All the cheats/scripts/injectors/kacktools get flagged by virustotal.

**Comment by akiii-afk** [comment]
why would I lie. What incentive do I have in lying LOL. If it works I am perfectly happy endorsing it. Just like the original Renz skin. That product was flagged but, I never got hacked or got viruses. Like obviously if I got hacked and this was the only thing flagged by malwarebytes on my pc, than its likely the cause, seeing as its the only thing I have downloaded this past month.

**Comment by mrlobaloba420** [comment]
this is a troll post, if you dont know how to read a source code, and you are paranoid, just dont use it, its that simple really. The source code exists for us to read and you know the owner of the program is pretty transparent of his program.

**Comment by Bullseye110** [comment]
> why would I lie. What incentive do I have in lying LOL. If it works I am perfectly happy endorsing it. Just like the original Renz skin. That product was flagged but, I never got hacked or got viruses. Like obviously if I got hacked and this was the only thing flagged by malwarebytes on my pc, than its likely the cause, seeing as its the only thing I have downloaded this past month.

You were hacked from something else, maybe some chrome vulnerabilities.

**Comment by emlautarom1** [comment]
> this is a troll post, if you dont know how to read a source code, and you are paranoid, just dont use it, its that simple really. The source code exists for us to read and you know the owner of the program is pretty transparent about it.

This is not true. The binaries in the `Release` page are different from what you can get by building from source, and the later does not work. There is quite the discrepancy and the only way to learn about what the code does is to put the `.dll` through a disassembler.

**Comment by am1guard** [comment]
mine hacked 2 like 1 mounths ago. like every accounts i have like twitter telegram discord steam insta face everything hacked. i even couldnt factory reset my pc. it was a nightmare
